### ISSUE TITLE
Part summary button

### ISSUE THREAD

**Issue by merkelp** [puhep/pudb#79]
Hello,
when I am on the detailed test results page for a module, the part summary button is on the bottom of the page, can we move it to the top?

Thanks,
Petra

**Comment by NickHinton** [comment]
Hi Petra,

Which page are you talking about, exactly? Is it the full test summary page, something like:
http://www.physics.purdue.edu/cmsfpix/Submission_p/summary/summaryFull.php?name=P-A-3-45 
If so, that page already has a back to summary button on top. 

If you could provide a link to the page in question that would be helpful.

Thanks,
Nick

**Comment by merkelp** [comment]
Hi Nick,
yes, after submitting this request I noticed that some modules do have 
this, but I found several that didn't.
Let me find some examples. Maybe they are just too old modules and I 
should forget about it, i it's fixed for all new modules.

Hmm, naturally I can not find the examples anymore. Maybe it's better to 
close this issue for now, maybe I was imagining things early this morning.

Petra
